test/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/NVIDIA/nvidia-container-toolkit/test
```

Comment:
not critical to this PR, but we should ensure that we update the dependabot configs.

Reply:
Was it done here?